### PR TITLE
Implemented the share button

### PR DESCRIPTION
**Pull request by sai-krishna-git**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -55,9 +55,24 @@ function App() {
...
+        console.log("Link shared Successfully");
+    })
+      .catch((error) => {
+        console.log("Erorr Sharing: ",error);
+      });
+    }
+    else{
+      alert("Your browser doesn't support the Web Share API");
+    }
+  };
```
Comment (Sarath191181208) [inline src/App.jsx]: rather than the user getting an alert try to copy the url and alert the user that the url has been coiped to clipboard

**Review by Sarath191181208 — COMMENTED** [review]
Try to implement these changes

**Comment by Sarath191181208** [comment]
Sorry, but if misled you but this is what I had in mind:
if the `navigator.share` doesn't exists or errored
then, copy to clipboard

**Comment by sai-krishna-git** [comment]
Implemented the changes. Check now.

**Pull request merged**